File: backend/src/main/java/io/github/daegwonkim/backend/repository/RentalItemLikeRecordRepository.java
package io.github.daegwonkim.backend.repository;

import io.github.daegwonkim.backend.entity.RentalItemLikeRecord;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.UUID;

public interface RentalItemLikeRecordRepository extends JpaRepository<RentalItemLikeRecord, UUID> {
}

File: backend/domain/src/main/kotlin/io/github/daegwonkim/backend/repository/jpa/RentalItemLikeRecordRepository.kt
package io.github.daegwonkim.backend.repository.jpa

import io.github.daegwonkim.backend.entity.RentalItemLikeRecord
import org.springframework.data.jpa.repository.JpaRepository

interface RentalItemLikeRecordRepository : JpaRepository<RentalItemLikeRecord, Long> {
    fun deleteByUserIdAndRentalItemId(userId: Long, rentalItemId: Long)
}